src/scss/components/_helpers.scss

```diff
@@ -232,7 +314,7 @@
 
 .border,
 %border {
-  bottom: 1px solid $border-color;
+  border: 1px solid $border-color;
```

Comment:
this looked like it was a bad copy paste (as well as all the `no-border**` below)